src/CommunityToolkit/Xamarin.CommunityToolkit.Sample/Xamarin.CommunityToolkit.Sample.csproj

```diff
@@ -32,15 +32,19 @@
   </ItemGroup>
 
   <ItemGroup>
+    <EmbeddedResource Update="Views\BehaviorsPage.xaml">
```

Comment:
This change is not necessary.

Reply:
Thanks for pointing this out, I need to do another pass through the csproj files and make sure there aren't weird things like this that got in there

Reply:
Code push on the way, this is removed